src/components/FieldCheckboxes/index.tsx

```diff
@@ -72,7 +72,7 @@ const FieldCheckboxesContext = createContext<FieldCheckboxesContextProps>(
   {} as TODO
 );
 
-type FieldCheckboxesProps = FieldProps &
+type FieldCheckboxesProps = FieldProps<Array<Value>> &
```

Comment:
Just a little question about how TypeScript is working. Why using this syntax instead of `FieldProps<Value[]>` ? Maybe my suggestion doesn't work ?

Reply:
It's equivalent, it's an opinionated choice. We use it because it's more readable (you read "it's an array of Value", and not "it's a Value in array")